lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -208,6 +208,20 @@ function wpcom_custom_error_handler( $whether_i_may_die, $type, $message, $file,
 		error_log( sprintf( '%s: %s in %s on line %d [%s] [%s]', $string, $message, $file, $line, $source, $backtrace ) );
 	}
 
+	if ( function_exists( 'do_action' ) ) {
+		/**
+		 * Fires when a PHP error occurs.
+		 *
+		 * @since 2.1.0
```

Comment:
What is 2.1.0? :-) I mean, how do we version mu-plugins?

Reply:
a copy-pasta 🍝